bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java

```diff
@@ -667,4 +675,9 @@ public static String getCurrentTarget()
 		}
 		return new SDKConfigJsonReader(project).getValue("IDF_TARGET"); //$NON-NLS-1$
 	}
+	
+	public static String getParseableVarValue(String var)
+	{
+		return "{" + var + "}"; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
We can append the $ sign to follow the standard convention with other Eclipse IDE variables